lib/msal-core/src/Utils.ts

```diff
@@ -453,6 +454,19 @@ export class Utils {
     return scopes.filter(value => value !== scope);
   }
 
+  /**
+   * Append extraScopesToConsent if passed in login request
+   * @param scopes
+   * @param extraScopesToConsent
+   */
+  static appendScopes(scopes: Array<string>, extraScopesToConsent: Array<string>) : Array<string> {
```

Comment:
Yep. I wanted room to confirm this functionality because though extraScopesToConsent is not much work now, I do know that some changes may come. Well I guess for now I can avoid a function call.

Reply:
Changes made.

Reply:
Pushed now @manoj-rath, Please review.

Reply:
Changes made.